src/components/unit.tsx

```diff
@@ -34,21 +36,18 @@ const UnitStatus = (props: unitProps) => {
     <>
       {currentStatus !== STATUS_CODES.NOT_HOME && <>{status}</>}
       {currentStatus === STATUS_CODES.NOT_HOME && (
-        <Badge bg="secondary" text="light" className="me-1">
-          {status} {nhcount}
-        </Badge>
+        <span className="container-nothome me-1">
+          <Image fluid src={envelopeImage} />
+          <div className="badge-nothome">{nhcount}</div>
```

Comment:
![Screenshot 2022-11-22 at 10 44 58 AM](https://user-images.githubusercontent.com/40650158/203208661-88feaec0-1af7-45c8-a261-3dda35fdbb03.png)

There is a translucent border on the count.
Can i suggest the following css to achieve translucency on the envelope instead?
```
.badge-nothome {
  color: black;
  background-color: #ffffff80; <- no need for the transparent box.
  position: absolute;
  top: 20%; <- may have to adjust this slightly as the count is touching the top border of the envelope.
  left: 20%;
  font-size: 0.8em;
  width: 1em;
}

. nothome-envelope {
  opacity: 0.75; <- can play around with the value to achieve the optimal translucency 
}
```

The rest of the changes looks fine. Nice work.

Reply:
I forgot to mention that you can also increase the font weight of the count. 

`font-weight: bold;` <- can adjust so that its clear enough

Reply:
![Screenshot 2022-11-22 at 5 25 10 PM](https://user-images.githubusercontent.com/40650158/203276593-ffc00cc7-6c17-44eb-9112-96f961d77827.png) 

Looks great. I'll proceed to merge your changes.